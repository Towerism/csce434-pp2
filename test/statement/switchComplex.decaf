int asdf() {
    switch(1) {
    case 1:
         qwer;
         1;
         break;
    case 2:
    case 3:
    case 4:
         5;
         rewq;
    case 5:
    default:
         1234;
         rewq;
         qwer;
         fdsa;
    }
}