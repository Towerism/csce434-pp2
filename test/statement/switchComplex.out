
   Program: 
  1   FnDecl: 
         (return type) Type: int
  1      Identifier: asdf
         (body) StmtBlock: 
            SwitchStmt: 
  2            (test) IntConstant: 1
               CaseStmt: 
  3               (label) IntConstant: 1
  4               FieldAccess: 
  4                  Identifier: qwer
  5               IntConstant: 1
  6               BreakStmt: 
               CaseStmt: 
  7               (label) IntConstant: 2
               CaseStmt: 
  8               (label) IntConstant: 3
               CaseStmt: 
  9               (label) IntConstant: 4
 10               IntConstant: 5
 11               FieldAccess: 
 11                  Identifier: rewq
               CaseStmt: 
 12               (label) IntConstant: 5
               (default) DefaultStmt: 
 14               IntConstant: 1234
 15               FieldAccess: 
 15                  Identifier: rewq
 16               FieldAccess: 
 16                  Identifier: qwer
 17               FieldAccess: 
 17                  Identifier: fdsa
